crates/sui-indexer-alt-jsonrpc/src/lib.rs

```diff
@@ -193,6 +205,7 @@ impl Default for RpcArgs {
         Self {
             rpc_listen_address: "0.0.0.0:6000".parse().unwrap(),
             max_in_flight_requests: 2000,
+            slow_request_threshold_ms: 60_000,
```

Comment:
We may want to lower this threshold but I'm fine with this since it should be easily changeable on sui-operations side.